gemfiles/rails_master.gemfile

```diff
@@ -2,17 +2,11 @@
 
 source "https://rubygems.org"
 
-gem "rails", git: 'https://github.com/rails/rails'
-gem "arel", git: 'https://github.com/rails/arel'
-
+gem "rails", git: "https://github.com/rails/rails"
 gem "appraisal"
 gem "bcrypt", "~> 3.1", require: false
 gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
-gem "sqlite3", "~> 1.3", "< 1.4", platform: [:ruby, :mswin, :mingw, :x64_mingw]
+gem "sqlite3", "~> 1.4", platform: [:ruby, :mswin, :mingw, :x64_mingw]
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw]
 
-%w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
```

Comment:
Latest stable RSpec supports Rails 6?

Reply:
I'm working on let Doorkeeper support Rails 6, currently Rails 6 will fail on other part.

Remove this because the file `# This file was generated by Appraisal`

Reply:
Just confirmed, latest stable RSpec supports Rails 6